seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -143,6 +148,77 @@
             "total_entries": 6
         }
 
+PROJECT_GUID = 'R0001_1kg'
+ADD_DATA_UPDATED_ANVIL_VARIABLES = {
+    "key": "AnVIL_WES",
+    "value": json.dumps({
+        "active_projects": [PROJECT_GUID],
+        "vcf_path": "gs://test_bucket/test_path.vcf",
+        "project_path": "gs://seqr-datasets/v02/GRCh37/AnVIL_WES/{guid}/v1".format(guid=PROJECT_GUID),
+        "projects_to_run": [PROJECT_GUID] })
+}
+ADD_DATA_UPDATE_DAG_TASKS_RESP = {
```

Comment:
Maybe these new test codes are unnecessary because they are testing the identical codes as creating a new Anvil project. What do you think?